### PR TITLE
shader model 6.6 docs

### PR DESCRIPTION
**Pull request by BitMD**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ These specs are presented with different interfaces at two different URLs:
     * [Shader Model 6.3](https://github.com/microsoft/DirectXShaderCompiler/wiki/Shader-Model-6.3)
     * [Shader Model 6.4](https://github.com/microsoft/DirectXShaderCompiler/wiki/Shader-Model-6.4)
     * [Shader Model 6.5](d3d/HLSL_ShaderModel6_5.md)
+    * [Shader Model 6.5](d3d/HLSL_ShaderModel6_6.md)
```
Comment (jenatali) [inline README.md:52]: Text for this says 6.5.

**Review by jenatali — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BitMD) [reply README.md]: Fixed.

File: d3d/HLSL_ShaderModel6_6.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+CBV_SRV_UAV heap or the Sampler heap. No root signature descriptor table mapping is required for this resource
+creation method, but new global root signature flags are used to indicate the use of each heap from the shader.
+
+A short example is below.
+```C++
+Texture2D<float4> myTexture = ResourceDescriptorHeap[texIdx];
+float4 result = myTexture.Sample(SamplerDescriptorHeap[sampIdx], coord);
+```
+
+See the [Shader Model 6.6 Dynamic Resources](SM_6_6_DynamicResources.md) documentation for more details.
```
[the remark this answers is not in the document]
Comment (BitMD) [reply d3d/HLSL_ShaderModel6_6.md]: fixed

File: d3d/HLSL_SM_6_6_Int64_and_Float_Atomics.md [hunk 3]
```diff
@@ -0,0 +1,629 @@
...
+ to render more realistic scenes with greater performance in a variety of ways.
+
+By adding support for 64-bit integer and bitwise floating-point values to these operations,
+original new rendering methods and optimizations become possible.
+
+---
+
+# Contents
+
+- [New Atomic Functions](#new-atomic-functions)
```
[the remark this answers is not in the document]
Comment (BitMD) [reply d3d/HLSL_SM_6_6_Int64_and_Float_Atomics.md:43]: fixed

File: d3d/HLSL_ShaderModel6_6.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+
+
+<a id="pack-and-unpack"></a>
+# Pack and Unpack Intrinsics
+
+A new set of intrinsics are being added to HLSL for processing of packed 8bit data such as colors. New packed datatype 
+are also added to HLSL's front end to symbolize a vector of packed 8bit values. 
+
+See the [Pack/Unpack Math Intrinsics](HLSL_SM_6_6_Pack_Unpack_Intrinisics.md) documenation for more details.
```
[the remark this answers is not in the document]
Comment (BitMD) [reply d3d/HLSL_ShaderModel6_6.md]: fixed

File: d3d/HLSL_SM_6_6_DynamicResources.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+In HLSL, the CBV/SRV/UAV Descriptor Heap is referred to as the Resource Descriptor Heap.
+
+Additional changes to existing handle patterns
+are introduced in this shader model
+to unify code paths and remove unnecessary metadata.
+
+## Contents
+
+- [HLSL Create Resource by Indexing Directly into Heap](#hlsl-create-resource-by-indexing-directly-into-heap)
```
Comment (jenatali) [inline d3d/HLSL_SM_6_6_DynamicResources.md:26]: One more dead link :)

**Review by jenatali — APPROVED** [review]
Looks good to me know.

**Pull request merged**